### ISSUE TITLE
Paypal error: "Sorry - your action could not be completed"

### ISSUE THREAD

**Issue by jackkinsella** [spree-contrib/spree_paypal_express#135]
If anyone's Spree website suddenly stopped accepting Paypal payments today and after redirection to Paypal they saw the error in this issue's title then this failure is due to a bug on Paypal's end. I've talked to their technical support and they've filed a ticket. For now the (tested) hot fix is the following:

```
Spree::CheckoutController.class_eval do
  # "en_US" supposedly works too
  def user_locale
      "GB"
  end
end
```

**Comment by dashdotat** [comment]
Yep, bitten with the same thing - user_locale is currently returning "en" (which is incorrect) and should be "GB" for the UK - I think a patch to allow setting locale for PayPal in the configuration is needed (if I get a chance I'll try and get a pull-request for this at some point)
